runtime/dali/src/governance.rs

```diff
@@ -0,0 +1,152 @@
+	type WeightInfo = weights::membership::WeightInfo<Runtime>;
+}
+
+impl collective::Config<NativeTechnicalMembership> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = collective::PrimeDefaultVote;
+	type WeightInfo = weights::collective::WeightInfo<Runtime>;
+}
+
+// NOTE: this is for testing runtime to fast track
+parameter_types! {
+	pub const CooloffPeriod: BlockNumber = 7 * DAYS;
+
+	pub const LaunchPeriod: BlockNumber = HOURS;
+	pub const VotingPeriod: BlockNumber = 5 * DAYS;
+	pub const FastTrackVotingPeriod: BlockNumber = HOURS;
+	pub MinimumDeposit: Balance = 100 * CurrencyId::unit::<Balance>();
+	pub const EnactmentPeriod: BlockNumber = HOURS;
+	// Note that Kusama uses 10 millis, however KSM is significantly more expensive
+	// https://github.com/paritytech/polkadot/blob/dc784f9b47e4681897cfd477b4f0760330875a87/runtime/kusama/src/lib.rs#L237
+	// so we increase it by a factor 10. This might still be on the low side.
+	pub PreimageByteDeposit: Balance = CurrencyId::milli::<u128>() * 100_u128;
+	pub const InstantAllowed: bool = true;
+	pub const MaxVotes: u32 = 100;
+	pub const MaxProposals: u32 = 100;
+	pub const DemocracyId: LockIdentifier = *b"democrac";
+	pub RootOrigin: Origin = frame_system::RawOrigin::Root.into();
+}
+
+impl democracy::Config<NativeDemocracy> for Runtime {
+	type Proposal = Call;
+	type Event = Event;
+	type Currency = Balances;
+	type EnactmentPeriod = EnactmentPeriod;
+	type LaunchPeriod = LaunchPeriod;
+	type VotingPeriod = VotingPeriod;
+	type VoteLockingPeriod = EnactmentPeriod;
+	type MinimumDeposit = MinimumDeposit;
+
+	type ExternalOrigin = EnsureRootOrHalfNativeCouncil;
+	type ExternalMajorityOrigin = EnsureRootOrHalfNativeCouncil;
+	type ExternalDefaultOrigin = EnsureRootOrHalfNativeCouncil;
+
+	type FastTrackOrigin = EnsureRootOrHalfNativeTechnical;
+	type InstantOrigin = EnsureRootOrHalfNativeTechnical;
+	type InstantAllowed = InstantAllowed;
+
+	type FastTrackVotingPeriod = FastTrackVotingPeriod;
+
+	type CancellationOrigin = EnsureRootOrAllNativeTechnical;
+
+	type BlacklistOrigin = EnsureRootOrHalfNativeCouncil;
+	type CancelProposalOrigin = EnsureRootOrHalfNativeCouncil;
+	type VetoOrigin = EnsureNativeTechnicalMember;
+	type OperationalPreimageOrigin = EnsureNativeCouncilMember;
+	type Slash = Treasury;
+
+	type CooloffPeriod = CooloffPeriod;
+	type MaxProposals = MaxProposals;
+	type MaxVotes = MaxVotes;
+	type PalletsOrigin = OriginCaller;
+
+	type PreimageByteDeposit = PreimageByteDeposit;
+	type Scheduler = Scheduler;
+	type WeightInfo = weights::democracy::WeightInfo<Runtime>;
+	type EnsureRoot = EnsureRoot<AccountId>;
+	type DemocracyId = DemocracyId;
+	type Origin = Origin;
+	type ProtocolRoot = RootOrigin;
+}
+
+// NOTE: making it multi via module_cdp_treasury seems fails other pallets
```

Comment:
likely yes. let me recheck. problem is that some pallets in deep depend on single instance of treasury some how. and fail to compile. will recheck

Reply:
```
  [0m   [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: `#[warn(unused_imports)]` on by default[0m

  [0m[1m[33mwarning[0m[0m[1m: unused import: `Permill`[0m
  [0m  [0m[0m[1m[38;5;12m--> [0m[0m/home/dz/git/blockchain/polkadot/composable/frame/liquidations/src/lib.rs:75:34[0m
  [0m   [0m[0m[1m[38;5;12m|[0m
  [0m[1m[38;5;12m75[0m[0m [0m[0m[1m[38;5;12m| [0m[0m    use sp_runtime::{DispatchError, Permill, Perquintill};[0m
  [0m   [0m[0m[1m[38;5;12m| [0m[0m                                    [0m[0m[1m[33m^^^^^^^[0m

  [0m[0m[1m[33mwarning[0m[1m:[0m `pallet-liquidations` (lib) generated 2 warnings
  [0m[0m[1m[32m   Compiling[0m ibc-runtime-api v0.1.0 (/home/dz/git/blockchain/polkadot/composable/frame/ibc/ibc-runtime-api)
  [0m[0m[1m[32m   Compiling[0m dali-runtime v0.1.0 (/home/dz/git/blockchain/polkadot/composable/runtime/dali)
  [0m[1m[38;5;9merror[E0277][0m[0m[1m: the trait bound `Runtime: pallet_treasury::Config<Instance1>` is not satisfied[0m
  [0m    [0m[0m[1m[38;5;12m--> [0m[0m/home/dz/git/blockchain/polkadot/composable/runtime/dali/src/xcmp.rs:217:17[0m
  [0m     [0m[0m[1m[38;5;12m|[0m
  [0m[1m[38;5;12m217[0m[0m  [0m[0m[1m[38;5;12m| [0m[0m    type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;[0m
  [0m     [0m[0m[1m[38;5;12m| [0m[0m                   [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m[0m [0m[0m[1m[38;5;9mthe trait `pallet_treasury::Config<Instance1>` is not implemented for `Runtime`[0m
  [0m     [0m[0m[1m[38;5;12m|[0m
  [0m     [0m[0m[1m[38;5;12m= [0m[0m[1mhelp[0m[0m: the following implementations were found:[0m
  [0m               <Runtime as pallet_treasury::Config<Instance2>>[0m
  [0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: required because of the requirements on the impl of `Callable<Runtime>` for `pallet_treasury::Pallet<Runtime, Instance1>`[0m
  [0m[1m[38;5;10mnote[0m[0m: required because it appears within the type `Call`[0m
  [0m    [0m[0m[1m[38;5;12m--> [0m[0m/home/dz/git/blockchain/polkadot/composable/runtime/dali/src/lib.rs:1076:1[0m
  [0m     [0m[0m[1m[38;5;12m|[0m
  [0m[1m[38;5;12m651[0m[0m  [0m[0m[1m[38;5;12m| [0m[0m [0m[0m[1m[38;5;10m|[0m[0m [0m[0mparameter_types! {[0m
```
very cryptic